tests/lib/utils/ember-test.js

```diff
@@ -492,13 +569,58 @@ describe('isComputedProp', () => {
 
 describe('isObserverProp', () => {
   let node;
+  let context;
 
   it("should check if it's an observer prop", () => {
-    node = parse('observer()');
+    context = new FauxContext(`
+      export default Controller.extend({
+        someObserver: observer(),
+      });
+    `);
+    node = context.ast.body[0].declaration.arguments[0].properties[0];
+    expect(emberUtils.isObserverProp(node)).toBeTruthy();
+
+    context = new FauxContext(`
+      export default Controller.extend({
+        someObserver: Ember.observer(),
+      });
+    `);
+    node = context.ast.body[0].declaration.arguments[0].properties[0];
+    expect(emberUtils.isObserverProp(node)).toBeTruthy();
+
+    context = new FauxContext(`
+      export default Component.extend({
+        levelOfHappiness: observer("attitude", "health", () => {
+        }),
+        vehicle: alias("car")
```

Comment:
Is the `vehicle` prop here necessary for the test?

Reply:
🤦‍♀ Nope! Copy-pasta. Will update